core/templates/dev/head/admin/admin.html

```diff
@@ -11,93 +11,203 @@
-        Reload All Explorations
-      </button>
+        {% endif %}
+      </div>
+    </nav>
+    <div align="center" ng-if="message" style="background-color: #ffffff; position: fixed; max-width: 30%; z-index: 3000; border: 1px solid #00376d; right: 30px; bottom: 30px">
+      <[message]>
     </div>
-
     <br>
-
-    <h3>Reload a single collection</h3>
-    <div class="container" style="margin-left: 0;">
-      {% for collection in demo_collections %}
-        <div class="row protractor-test-reload-collection-row">
-          <span class="col-lg-4 col-md-4 col-sm-4 protractor-test-reload-collection-title">
-            {{collection[1]}}
-          </span>
-          <span class="col-lg-2 col-md-2 col-sm-2">
-            <button type="button" class="btn btn-default protractor-test-reload-collection-button" ng-click="reloadCollection({{collection[0]}})">
+    <div class="row">
+    <div class="col-lg-12 col-md-12 col-sm-12" ng-show='activities' style="margin-top: 16px;">
+      <md-card class="oppia-page-card oppia-long-text" style="max-width:700px">
+        <div class="container" style="margin-left: 0;">
+          <h3>Reload a single exploration</h3>
+          {% for exploration in demo_explorations %}
+          <div class="row protractor-test-reload-exploration-row">
+            <span class="col-lg-4 col-md-4 col-sm-4 protractor-test-reload-exploration-title">
+              {{exploration[1]}}
+            </span>
+            <span class="col-lg-2 col-md-2 col-sm-2">
+              <button type="button" class="btn btn-default protractor-test-reload-exploration-button pull-right" ng-click="reloadExploration({{exploration[0]}})">
               Reload
-            </button>
-          </span>
-        </div>
-      {% endfor %}
-    <h3>Jobs</h3>
-
-    <h4>Current time: {{human_readable_current_time}}</h4>
-
-    <h4>One-off jobs</h4>
-    <ul>
-      {% for job_spec in one_off_job_specs %}
-        <li>
-          {{job_spec.job_type}}
-          <button ng-click="startNewJob('{{job_spec.job_type}}')">
-            Start new job
+              </button>
+            </span>
+          </div>
+          {% endfor %}
+          <button type="button" class="btn btn-default protractor-test-reload-all-explorations-button" ng-click="reloadAllExplorations()">
+          Reload All Explorations
           </button>
-          {% if job_spec.is_queued_or_running %}
-            <span class="oppia-serious-warning-text">
-              <strong>(warning: an instance of this job type is already running)</strong>
+        </div>
+      </md-card>
+      <md-card class="oppia-page-card oppia-long-text" style="max-width: 700px">
+        <div class="container" style="margin-left: 0;">
+          <h3>Reload a single collection</h3>
+          {% for collection in demo_collections %}
+          <div class="row protractor-test-reload-collection-row">
```

Comment:
This should be indented by 2 (have a look at how it was done before). In this codebase, we optimize the formatting for people reading the code, not for the HTML that is displayed in the frontend. People look at the former more often.